services/src/test/java/org/exoplatform/addons/gamification/rest/TestAnnouncementRest.java

```diff
@@ -194,7 +195,7 @@ public void testGetAllAnnouncementByChallenge() throws Exception {
     Challenge challenge = new Challenge(0,
                                         "update challenge",
                                         "challenge description",
-                                        1l,
+                                        1L,
```

Comment:
just for curiosity and learning, why this change ?

Reply:
No practical difference, it's just to remove the editor's warning and eliminate the possibility of confusion between a lowercase **l** and the number **1**